### PR TITLE
Add image previews

### PR DESCRIPTION
**Pull request by hoangnv-bkhn**
image

### CODE REVIEW

**Comment by hoangnv-bkhn** [comment]
ready

File: app/views/shared/_micropost_form.html.erb [hunk 1]
```diff
@@ -5,6 +5,8 @@
   </div>
   <%= f.submit t(".post"), class: "btn btn-primary" %>
   <span class="image">
-    <%= f.file_field :images, :multiple => true, accept: Settings.micropost.accept_path %>
+    <%= f.file_field :images, :onchange => "previewFiles(#{Settings.micropost.preview_size})", :multiple => true, accept: Settings.micropost.accept_path %>
   </span>
+  <div id="preview"></div>
 <% end %>
+<%= javascript_pack_tag 'micropost' %>
```
Comment (ghost) [inline app/views/shared/_micropost_form.html.erb]: import vào file application.js đi em

**Comment by hoangnv-bkhn** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**